service/share/dbus.conf

```diff
@@ -23,12 +25,24 @@
     <allow send_destination="org.opensuse.DInstaller.Language" />
     <allow receive_sender="org.opensuse.DInstaller.Language" />
 
+    <allow send_destination="org.opensuse.DInstaller.Questions" />
+    <allow send_destination="org.opensuse.DInstaller.Questions"
+           send_interface="org.freedesktop.DBus.Introspectable"/>
+    <allow send_destination="org.opensuse.DInstaller.Questions" />
```

Comment:
Hm, this line is duplicate, overall this is equivalent to

    <allow send_destination="org.opensuse.DInstaller.Questions" />
    <allow   receive_sender="org.opensuse.DInstaller.Questions" />